contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/advisors/DefaultSiteAdvisor.java

```diff
@@ -33,6 +34,12 @@ public boolean siteCanUseLTIReviewService(Site site) {
 		return true;
 	}
 	
+	@Override
+	public boolean siteCanUseLTIReviewServiceForAssignment(Site site, Date assignmentCreationDate)
```

Comment:
Passing in the assignment adds another dependency on the Assignments tool, which is probably not the direction we want to go in. Also, the way I see these advisors is that they are a temporary migration aid that will be removed once the original Turnitin API is disabled. Once there is no longer a choice between legacy or LTI, there won't be a need for them. Even if they do stick around, this method certainly won't.